src/Build.UnitTests/BackEnd/SdkResolverService_Tests.cs

```diff
@@ -43,18 +47,20 @@ public SdkResolverService_Tests()
 
         public void Dispose()
```

Comment:
please observe the [best practice implementation of the disposable pattern] (https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/dispose-pattern)

Reply:
I looked at the s_sdkResolverService and I can see only my test using it. There might have been some older commit doing that due to an imprecise refactoring.

Reply:
As for the Dispose(), after a second look, I've removed the remaining static instance and so removed the Dispose as well since it should no longer be necessary.